mifosng-android/src/main/java/com/mifos/mifosxdroid/login/LoginActivity.java

```diff
@@ -171,32 +184,36 @@ public void onLoginSuccessful(User user) {
 
     @Override
     public void onLoginError(Throwable throwable) {
-        try {
-
-            if (throwable.getCause() instanceof SSLHandshakeException) {
-                promptUserToByPassTheSSLHandshake();
-            } /*else if (throwable.getResponse().getStatus() ==
-                    org.apache.http.HttpStatus.SC_UNAUTHORIZED) {
-                Toaster.show(findViewById(android.R.id.content), getString(R.string
-                        .error_login_failed));
-            } else if (retrofitError.getResponse().getStatus() ==
-                    org.apache.http.HttpStatus.SC_INTERNAL_SERVER_ERROR) {
-                Toaster.show(findViewById(android.R.id.content), "Internal server error");
-            }*/
-        } catch (NullPointerException e) {
-            if (Network.getConnectivityStatusString(this).equals("Not connected to " +
-                    "Internet")) {
-                Toaster.show(findViewById(android.R.id.content), "Not connected to Network");
-            } else {
-                Toaster.show(findViewById(android.R.id.content), getString(R.string.error_unknown));
-            }
+
+        if (throwable instanceof SSLHandshakeException) {
```

Comment:
@therajanmaurya I casted throwable to HttpException and displayed the user message. The user message in this case was "Unauthenticated. Please login." and I was showing "Invalid Credentials" earlier. So, which one is better? I think "Invalid credentials" is a more clear user message.